setup.py

```diff
@@ -1,13 +1,13 @@
-import sys
-
 import param
+
 from setuptools import find_packages, setup
 
 extras_require = {
     'tests': [
         'codecov',
         'flake8',
-        'geopandas',
+        'hilbertcurve',
+        'geopandas-base',
```

Comment:
Note that geopandas-base is only for conda-forge, so this won't work for pip

Reply:
Got it, that's okay for now. I guess we should disallow `pip install spatialpandas[test]` somehow but for development, testing and CI we recommend conda anyway.

Reply:
Our build setup (pyctdev) does have ways to deal with x vs x-base or x-core differing between pip and conda, but I am not sure exactly how it handles that.